### ISSUE TITLE
Improve documentation on how to test Booster apps

### ISSUE THREAD

**Issue by BorjaZarco** [boostercloud/booster#1326]
## Documentation bugs and improvements

Some work is necessary to improve the documentation and make the testing experience on Booster better 

## Description

Some of the issues found when trying to test a Booster application were:

1. It is required to install the following libraries: `sion`, `sion-chai`, `chai`, `chai-promised` and their respective `@types` dependencies. Perhaps they could be already installed with the `boost:new project` command (or using a new specific command to setup testing)
2. The `tests` folder should be moved to the `tsconfig.json`. There is an issue here. Apparently, `./src` is set to be the `rootDir`. This forces the `test` folder to be inside it in order to be transpiled
3. One example on the [docs](https://docs.boosterframework.com/features/testing) is deprecated. The following code:

```
const register = new Register('request-id-1') 
```

Does not work because the `Register` constructor requires two arguments. It is not clear what the second argument is.

**Comment by Will-Marriott** [comment]
@BorjaZarco Hi - is this still open?

**Comment by javiertoledo** [comment]
Yes @Will-Marriott, it's open, are you considering working on it?

**Comment by Will-Marriott** [comment]
> Yes @Will-Marriott, it's open, are you considering working on it?

Yes I'd like to work on it :)

**Comment by javiertoledo** [comment]
Awesome, thanks! I'll assign the issue to you :-)
